gov2/cloudfront/README.md

```diff
@@ -0,0 +1,79 @@
+# CloudFront code examples for the SDK for Go V2
+
+## Overview
+
+Shows how to use the AWS SDK for Go V2 to work with Amazon CloudFront.
+
+<!--custom.overview.start-->
+<!--custom.overview.end-->
+
+_CloudFront speeds up distribution of your static and dynamic web content, such as .html, .css, .php, image, and media files._
+
+## ⚠ Important
+
+* Running this code might result in charges to your AWS account. For more details, see [AWS Pricing](https://aws.amazon.com/pricing/?aws-products-pricing.sort-by=item.additionalFields.productNameLowercase&aws-products-pricing.sort-order=asc&awsf.Free%20Tier%20Type=*all&awsf.tech-category=*all) and [Free Tier](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=*all&awsf.Free%20Tier%20Categories=*all).
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+<!--custom.important.start-->
+<!--custom.important.end-->
+
+## Code examples
+
+### Prerequisites
+
+For prerequisites, see the [README](../README.md#Prerequisites) in the `gov2` folder.
+
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+<!--custom.examples.start-->
```

Comment:
This CloudFront example isn't in the meta (by design), so maybe we should add it to custom here for the time being?

Reply:
Could you propose what you want me to put in this block, then? (Sorry, just not familiar enough with it to write it)